python/cli/prelude_cli/views/partner.py

```diff
@@ -129,3 +121,98 @@ def partner_advisories(controller, partner, start, offset, limit):
     """ Get advisories provided by partner """
     with Spinner(description='Getting partner advisories'):
         return controller.list_advisories(partner=Control[partner], start=start, offset=offset, limit=limit)
+
+## --------- SCM Commands --------- ##
```

Comment:
is it too big of a change to make a new scm view and new export view? we're kind of straying away from the /{route}/ -> {route}_controller.py -> views/{route}.py pattern

Reply:
yeah, a new scm controller / cli module is my end goal too. I was just waiting for your PR to go in first and then I'd shift em all over (or you might in your PR). Just separated them out in this PR to make the shift easier. Guess I could also just make the new files now tho